service/StudentService.java

```diff
@@ -0,0 +1,51 @@
+package service;
+
+import dao.Dao;
+import model.Student;
+
+import java.util.Iterator;
+
+public class StudentService {
+    private final Dao<Student> studentDao;
+
+    public StudentService(Dao<Student> studentDao){
+        this.studentDao = studentDao;
+    }
+
+    public Student getStudentById(String id){
+        Student student = studentDao.get(id).get();
+        return student;
+    }
+
+    public void getAllStudents(){
+        System.out.println();
```

Comment:
`System.out.println()` belongs to the (UI) presentation layer! Do not use this inside the service layer.